docker-compose.yml

```diff
@@ -37,8 +37,12 @@ services:
     env_file: .env
   frontend:
     container_name: frontend
+    volumes:
+      - './app:/usr/app'
+      - '/usr/app/.next'
```

Comment:
Why are you mounting a volume to `/usr/app/.next` that is available on the host? This would just be available at `./app/.next` by the volume on line 40.

Reply:
I just followed the steps in this tutorial https://dev.to/kumareth/next-js-docker-made-easy-2bok which showed to include the .next and node_modules in the volumes list.